ddtrace/tracer/tracer.go

```diff
@@ -55,13 +63,21 @@ type tracer struct {
 
 	// pid of the process
 	pid string
+
+	// These integers track metrics about spans and traces as they are started,
+	// finished, and dropped
+	spansStarted, spansFinished, tracesDropped int64
 }
 
 const (
 	// flushInterval is the interval at which the payload contents will be flushed
 	// to the transport.
 	flushInterval = 2 * time.Second
 
+	// statsInterval is the interval at which span statistics will be sent with the
```

Comment:
Let's try and use the "health metrics" terminology throughout for consistency.
```suggestion
	// statsInterval is the interval at which health metrics will be sent with the
```

Reply:
👍